### ISSUE TITLE
`catalog` is not recognized in `fv3fit.train`

### ISSUE THREAD

**Issue by elynnwu** [ai2cm/fv3net#2379]
On HPC systems, we use a custom `--catalog` path for all diagnostics workflows. This should be expanded to training as well so that we can avoid reading from gcp (some nodes don't have internet access) and read directly from HPC storage.

**Comment by spencerkclark** [comment]
Indeed we thought about this some in #2204 — see in particular https://github.com/ai2cm/fv3net/issues/2204#issuecomment-1524037533.  We ended up going with the option of adding an attribute to the `BatchesFromMapperConfig` dataclass.  I use this for example [here](https://github.com/ai2cm/nudge-to-fine-stellar-workflow/blob/main/configs/templates/training-data-tendencies.yaml) in the nudge-to-fine-stellar-workflow.  

Does that also work for you?

**Comment by elynnwu** [comment]
Yes, I was not aware that this is an option. Thanks for pointing that out! Closing this issue now.
